packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -875,6 +889,10 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
     if (item.disabled || item.isDisabled) {
       return;
     }
+    if (this._processingTouch) {
+      return this._onItemClick(item, ev);
```

Comment:
can you add unit tests to validate the things you're changing?

Reply:
Will do!

Reply:
Added unit tests.